package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "clarity",
   "private": true,
-  "version": "0.6.14",
```

Comment:
AT some point we should start using the other parts of hte sesmantic version if the data contract breaks. This seems like it might be a 0.7.x release.

Reply:
It is backward compatible. We have been bumping minor version (0.7) when we make breaking changes.

Reply:
Can you clarify the "it is backward compataible" part? Can discuss in person.

Reply:
Discussed offline.